### PR TITLE
Add: created pull request template

### LINKED ISSUE #52: Add Pull Request Template
- use keywords to resolve issues (if applicable)
   - if the issue is not yet resolved but it's related, link the issue to the PR
- add screenshots if necessary
- link references or useful notes

related to `CONTRIBUTING.md`

### PR DESCRIPTION
**Pull request by Niravprajapati1**
Created pull request template 
Closes #52

### CODE REVIEW

**Comment by kachun333** [comment]
@Niravprajapati1 , you forget to use one of the keywords

image

Please have a look: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword

**Comment by Niravprajapati1** [comment]
You can review now

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+- [ ] My change requires changes to the documentation.
+
+- [ ] I have updated the documentation accordingly.
+
+- [ ] All new and existing tests passed.
+
+- [ ] This PR does not contain plagiarized content.
+
+- [ ] The title of my pull request is a short description of the requested changes.
```
Comment (kachun333) [inline .github/pull_request_template.md:37]: Not needed for this project 

```suggestion
```

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Screenshots
+
+<!-- Add all the screenshots which support your changes -->
+
+## Note to reviewers
+
+<!-- Add notes to reviewers if applicable -->
```
Comment (kachun333) [inline .github/pull_request_template.md:45]:
```suggestion
## Additional Notes

-
```

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+- [ ] All new and existing tests passed.
+
+- [ ] This PR does not contain plagiarized content.
+
+- [ ] The title of my pull request is a short description of the requested changes.
+
+## Screenshots
+
+<!-- Add all the screenshots which support your changes -->
```
Comment (kachun333) [inline .github/pull_request_template.md:41]:
```suggestion
## Screenshots

-
```

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+
+<!--
+
+[x] - Correct; marked as done
+
+[X] - Correct; marked as done
+
+[ ] - Not correct; marked as **not** done
+
+-->
```
Comment (kachun333) [inline .github/pull_request_template.md:23]:
```suggestion
## Describe Your Changes

-
```

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,45 @@
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
+
+## Fixes Issue
+
+<!-- Remove this section if not applicable -->
+
+<!-- Example: Closes #31 -->
```
Comment (kachun333) [inline .github/pull_request_template.md:7]:
```suggestion
## Fixed Issue(s)

- 
```

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,45 @@
+<!-- If your PR fixes an open issue, use `Closes #999` to link your PR with the issue. #999 stands for the issue number you are fixing -->
```
Comment (kachun333) [inline .github/pull_request_template.md:1]:
```suggestion
```

**Review by kachun333 — CHANGES_REQUESTED** [review]
hi @Niravprajapati1 , I proposed to keep the PR template simple

**Review by kachun333 — APPROVED** [review]

**Comment by kachun333** [comment]
thanks @Niravprajapati1

**Comment by Niravprajapati1** [comment]
No problem 👍

**Pull request merged**